spec/methods/auth/auth.ts

```diff
@@ -339,5 +363,175 @@ export namespace ConfirmPassReset {
    */
   export type Result = MessageResult;
 
+  export type Response = MessageResponse;
+}
+
+//////////////////////////////////
+// Configure MFA Types
+//////////////////////////////////
+
+/**
+ * Sets user MFA preferences
+ */
+export namespace SetMfaPreference {
+
+  export const HTTP:HttpMethods.POST = 'POST';
+  export const Path = `${authBasePath}/${ResourcePaths.configureMfa}`;
+
+  export interface Args {
+    smsMfaEnabled: boolean,
+    appMfaEnabled: boolean,
+    preferredMfa?: Challenges.MfaTypes 
+  }
+
+  /**
+   * Type guard; only returns true for valid `Args` objects.
+   * @param val 
+   */
+  export function isArgs(val:any): val is Args {
+    return keysAreBooleans(val, ['smsMfaEnabled', 'appMfaEnabled']) &&
+           (!val.preferredMfa || Challenges.isMfaTypes(val.preferredMfa));
+  }
+
+  /**
+   * Factory to produce an Args object with
+   * empty strings. Useful for getting the 
+   * correct shape as a value.
+   */
+  export function newArgs(): Args {
+    return {
+      smsMfaEnabled: false,
+      appMfaEnabled: false
+    }
+  }
+
+  /**
+   * When successful, the message ought to say something
+   * like, "An email has been sent with a temporary password."
+   */
+  export type Result = MessageResult;
+
+  export type Response = MessageResponse;
+}
+
+/**
+ * Setup Phone Number for SMS MFA
+ */
+export namespace SetupSmsMfa {
+
+  export const HTTP:HttpMethods.POST = 'POST';
+  export const Path = `${authBasePath}/${ResourcePaths.configureMfa}`;
+
+  export interface Args {
+    phoneNumber: string
+  }
+
+  /**
+   * Type guard; only returns true for valid `Args` objects.
+   * @param val 
+   */
+  export function isArgs(val:any): val is Args {
+    return keysAreStrings(val, ['phoneNumber']);
+  }
+
+  /**
+   * Factory to produce an Args object with
+   * empty strings. Useful for getting the 
+   * correct shape as a value.
+   */
+  export function newArgs(): Args {
+    return {
+      phoneNumber : ''
+    }
+  }
+
+  /**
+   * When successful, the message ought to say something
+   * like, "An email has been sent with a temporary password."
```

Comment:
Can you update this string to say what the message would actually be like?  Would it just say they've been sent a text?